### PR TITLE
OPT - Upload pypi package using Travis

### PR DESCRIPTION
**Pull request by CCardosoDev**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +1,29 @@
-language: python
-python:
-  - "3.7-dev"
-
 services:
-  - docker
+    - docker
```
Comment (mattiassluis) [inline .travis.yml]: Style: Looks like there are 2 too many spaces here?

File: .travis.yml [hunk 2]
```diff
@@ -1,16 +1,29 @@
...
+      install:
+        - pip install codecov
+      script:
+        - make test
+      after_success:
+        - codecov
+    - stage: publish
+      language: generic
+      deploy:
+          provider: script
```
Comment (mattiassluis) [inline .travis.yml]: Style: looks like this block is indented by 2 extra spaces as well

**Review by mattiassluis — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (CCardosoDev) [reply .travis.yml]: fixed

File: .travis.yml (hunk 1 above)
Comment (CCardosoDev) [reply .travis.yml]: fixed

**Review by mattiassluis — APPROVED** [review]

**Comment by mattiassluis** [comment]
LGTM

**Pull request merged**